qemu/tests/cfg/block_resize.cfg

```
@@ -14,6 +14,8 @@
         max_block_size = 2199023255552
         ovmf:
             max_block_size = 17592186044416
+        258_event_check = 'wevtutil qe Application /q:"*[System/EventID=258]" /f:text'
```

```suggestion
        check_258_event = 'wevtutil qe Application /q:"*[System/EventID=258]" /f:text'
```
Recommend the command parameter starts with the verb.

---

Get and thanks! Updated done~